ZHMModSDK/Src/ModSDK.cpp

```diff
@@ -1361,8 +1361,8 @@ struct EOS_Platform_Options {
     uint32_t TickBudgetInMilliseconds;
 };
 
-#define EOS_PF_LOADING_IN_EDITOR 0x00001
-#define EOS_PF_DISABLE_OVERLAY 0x00002
+constexpr auto EOS_PF_LOADING_IN_EDITOR = 0x00001;
+constexpr auto EOS_PF_DISABLE_OVERLAY = 0x00002;
```

Comment:
These and the above come from the EOS SDK, so they're better left as-is for reference's sake.

Reply:
Okay, sure thing - I'll return them back 👍